src/cheri_sys_regs.sail

```diff
@@ -65,6 +65,8 @@
 /* Capability control csr */
 
 bitfield ccsr : xlenbits = {
+  /* Bits allocated from 31 downwards are used for feature flags. */
+  tc      :  31,     /* tag-clearing error semantics */
   d       :  1,      /* dirty  */
```

Comment:
Do we also want to remove 'd' and 'e'?  I don't think CheriBSD has ever used them.  Not sure if we have any other software that ever has.

Reply:
That sounds good to me, although we should keep e until we have an implementation of the new semantics.